angular-workspace/projects/ni/nimble-angular/table-column/anchor/nimble-table-column-anchor-router-link-with-href.directive.ts

```diff
@@ -28,11 +31,54 @@ export class NimbleTableColumnAnchorRouterLinkWithHrefDirective extends RouterLi
             return;
         }
 
-        // Let the router handle this navigation
-        super.routerLink = href;
-        if (!this.onClick(clickEvent.button, clickEvent.ctrlKey, clickEvent.shiftKey, clickEvent.altKey, clickEvent.metaKey)) {
+        if (!this.handleLinkClick(href, clickEvent.button, clickEvent.ctrlKey, clickEvent.shiftKey, clickEvent.altKey, clickEvent.metaKey)) {
             clickEvent.preventDefault();
         }
-        super.routerLink = null;
+    }
+
+    private handleLinkClick(url: string, button: number, ctrlKey: boolean, shiftKey: boolean, altKey: boolean, metaKey: boolean): boolean {
```

Comment:
I can somewhat see how this implementation was adapted from the navigation tree component in systemlink-lib-angular, but it also doesn't look like it's handling everything that the nav tree does. I'm curious about the purpose of the divergence.